CIP-onchain-cert-metadata/README.md

```diff
@@ -0,0 +1,485 @@
+
+The metadata should be discoverable by all certification stakeholders, including end-users, DApp developers, and ecosystem components, such as light wallets and DApp stores. Information should be indexable by certification issuer, DApp developer, DApp and DApp version.
+
+### Use-cases and Stakeholders
+
+DApp developers will seek certification from one of the issuers, according to their desired level of certification and will refer to the certificate on several platforms (e.g. on their website or in their DApp registration on a DApp store).
+
+Certification is to be provided by certification issuers including: testing services (level 1), auditors (level 2), and verification services (level 3). Certification issuers will issue these certificates referring to a particular version of a DApp that has succesfully gone through a certification process.
+
+DApp stores and light wallets will be able to pull DApps information from DApps registration or chain exploring and will link a DApp to a corresponding set of certificates.
+
+End-user will interact with a DApp through a wallet and will be able to check the different certificates obtained by the DApp.
+
+## Specification
+
+### Definitions
+
+- **anchor** - A hash written on-chain that can be used to verify the integrity (by way of a cryptographic hash) of the data that is found off-chain.
+- **DApp** - A decentralised application that is described by the combination of metadata, certificate and a set of scripts.
+- **dApp Store** - A dApp aggregator application which follows on-chain data looking for and verifying dApp metadata claims, serving their users linked dApp metadata.
+- **Certification issuers** - A company that issues certification certificates on-chain.
+
+### Certification issuers
+
+Certification issuers will broadcast on-chain certificates that will represent th level of certification reached and present evidence of the work done.
+Certification issuers will sign the certificate to attest that they have done the work and prevent certificate forgeries.
+
+### Suggested validations
+
+- `integrity`: The DApp's metadata off-chain shall match the metadata **anchored** on-chain.
+- `trust`: The DApp's certification metadata transaction shall be signed by a trusted certfication issuer. This means a list of wallets of certification issuers needs to be known. It will then be up to the DApp store to decide which certification issuers are really trusted and publish a list of their own trusted certification issuers.
+
+### Certificate JSON Schema
+
+```json
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$title": "Certification Certificate", 
+    "type": "object", 
+    "properties": {
+        "subject": {
```

Comment:
Is there any correlation between this CIP-72's subject?

Reply:
Resolving since it seems clear from the following `description` field.  @matiwinnetou if you feel this is still an issue please unresolved & explain. 🤓